lib/Launcher.js

```diff
@@ -15,16 +15,18 @@
  */
 const os = require('os');
 const path = require('path');
-const removeSync = require('rimraf').sync;
+const {helper} = require('./helper');
+const rimrafAsync = helper.promisify(require('rimraf'));
```

Comment:
nit: let's name this `removeAsync`: the `rimraf` doesn't say much of what the function is doing.